alpha/lib/model/thumbAsset.php

```diff
@@ -42,4 +42,13 @@ public function getFinalDownloadUrlPathWithoutKs()
 						
 		return $finalPath;
 	}
+
+	public function keepOnEntryReplacement(entry $entry)
```

Comment:
Instead of passing the entry to this method you can simply use $entry = $this->getentry();